change/@fluentui-react-tree-7b614498-e79f-460b-a3a4-22ba4d9a2bea.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "tree stories and doc update",
+  "packageName": "@fluentui/react-tree",
+  "email": "petrduda@microsoft.com",
+  "dependentChangeType": "none"
+}
```

```suggestion
{
  "type": "prerelease",
  "comment": "tree stories and doc update",
  "packageName": "@fluentui/react-tree",
  "email": "petrduda@microsoft.com",
  "dependentChangeType": "patch"
}
```

let's make this a `prerelease` just to ensure storybook doc will be properly updated